frontend/src/components/Header.js

```diff
@@ -1,19 +1,19 @@
 import React from 'react';
 import styled from 'styled-components';
-import Logo from '../resources/medshare-logo.png';
+
```

Comment:
I think the `Header` component is no longer necessary now, instead you should directly use the `NavBar` component. The only reason we had this component was for holding the Medshare logo that was part of the old designs.

Reply:
dpme